app/src/server/modal-rpc/clients.ts

```diff
@@ -26,7 +26,7 @@ type ModalInput = {
   model: string;
   prompt: string;
   n?: number;
-  max_tokens?: number;
+  max_tokens?: number | null;
```

Comment:
Will Modal handle this properly by default?

Reply:
Good catch. No, vLLM throws an error. So inside `runInference` we should check if this key is `null` and remove it from the args before sending the request to Modal.

Reply:
Probably easier to not pass `max_tokens: null` to this function, and pass undefined instead.